tests/unit/amo/pages/TestAddon.js

```diff
@@ -1402,65 +1401,6 @@ describe(__filename, () => {
     });
   });
 
-  describe('mapStateToProps', () => {
```

Comment:
Note that all three of these cases are already addressed by other tests that exercise the `mapStateToProps` implicitly.